debian/changelog

```diff
@@ -0,0 +1,5 @@
+crs-scripts (0.1) stable; urgency=low
```

Comment:
I'd like a version number in the format 2020.10.19.1 as there is no really sensible major-minor versioning scheme for these scripts